llvm/lib/Transforms/Vectorize/VPlan.h

```diff
@@ -3938,6 +3941,22 @@ class VPlan {
   VPRegionBlock *getVectorLoopRegion();
   const VPRegionBlock *getVectorLoopRegion() const;
 
+  /// Get the vector early exit block
+  VPBasicBlock *getEarlyExit() {
```

Comment:
It looks like this is isn't needed any longer?

Reply:
Yep, removed thanks!